crates/oxc_transformer/src/lib.rs

```diff
@@ -202,6 +201,10 @@ impl<'a> VisitMut<'a> for Transformer<'a> {
         self.decorators.as_mut().map(|t| t.transform_declaration(decl));
     }
 
+    fn visit_jsx_opening_element(&mut self, elem: &mut JSXOpeningElement<'a>) {
```

Comment:
it seems that `visit_jsx_opening_element` will execute after `visit_expression`, so the `jsx_self` transformer will execute after `react_jsx` transformer, maybe i need to move  `self.jsx_self.as_mut().map(|t| t.transform_jsx_opening_element(elem));` into `visit_expression` ?